src/main/java/org/candlepin/model/OwnerProductCurator.java

```diff
@@ -1148,9 +1148,13 @@ public void clearProductEntityVersion(String productUuid) {
             return;
         }
 
+        String sql = "UPDATE " + Product.DB_TABLE + " SET entity_version = NULL " +
```

Comment:
This change also needs to occur in `OwnerContentCurator.clearContentEntityVersion`

Reply:
Good call! Updated `OwnerContentCurator.clearContentEntityVersion`